tests/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	cosmossdk.io/simapp v0.0.0-00010101000000-000000000000
 	github.com/cometbft/cometbft v0.37.0-rc3
 	github.com/cometbft/cometbft-db v0.7.0
-	github.com/cosmos/cosmos-sdk v0.47.0-rc2.0.20230220103612-f094a0c33410
+	github.com/cosmos/cosmos-sdk v0.47.0
```

Comment:
shouldnt this be `-rc3`?

Reply:
No because tests and simapp uses a replace, the version we input does not matter. For simapp version to show the proper SDK version, we simply just set the version of the release branch.
For instance, on main it is set to v0.48 (as it's the next version coming).